pkgs/games/anki/addons/ankimon/default.nix

```
@@ -0,0 +1,32 @@
+{
+  lib,
+  anki-utils,
+  fetchFromGitHub,
+  nix-update-script,
+}:
+anki-utils.buildAnkiAddon (finalAttrs: {
+  pname = "ankimon";
+  version = "1.288";
+  src = fetchFromGitHub {
+    owner = "Unlucky-Life";
+    repo = "ankimon";
+    tag = finalAttrs.version;
+    hash = "sha256-PPTmZpxeBv0zb8wbxLEozDdRJG+eYjMqGyx8y5na1MQ=";
+  };
+  passthru.updateScript = nix-update-script { };
+  meta = {
+    description = "Ankimon is a Anki Addon to Gamify your learning experience";
```

```suggestion
    description = "Anki Addon to Gamify your learning experience";
```
Not start with the package name